README.md

```diff
@@ -117,16 +122,11 @@ MariaDB cluster).
 - [FAQ](docs/src/faq.md)
 - [Blog](https://cloudnative-pg.io/blog/)
 
-### Useful links
+## Adopters
 
-- [Data on Kubernetes (DoK) Community](https://dok.community/)
-- ["Recommended Architectures for PostgreSQL in Kubernetes" by Gabriele Bartolini](https://www.cncf.io/blog/2023/09/29/recommended-architectures-for-postgresql-in-kubernetes/)
-- ["How CloudNativePG manages physical replication slots for PostgreSQL in Kubernetes" by Gabriele Bartolini](https://www.enterprisedb.com/blog/how-cloudnativepg-manages-replication-slots) (September 2023)
```

Comment:
This line has not been moved together with the others. Is it intentional?

Reply:
Yes. IMHO it is about a very specific topic for the main README.